apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-members/vo-members.component.ts

```diff
@@ -68,12 +75,19 @@ export class VoMembersComponent implements OnInit {
     this.membersService.findCompleteRichMembers(this.vo.id, this.searchString, this.attrNames).subscribe(
```

Comment:
This method now has hardcoded the statuses which are searched. You should add another argument for specifying the statuses that should be found. Then you can here search only members with statuses that are present in the select.